src/test/java/Pages/BasePage.java

```diff
@@ -0,0 +1,19 @@
+package Pages;
+
+import Drivers.Config;
+import Drivers.Driver;
+import lombok.SneakyThrows;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.PageFactory;
+
+public class BasePage {
+
+    final WebDriver driver;
+
+    @SneakyThrows
```

Comment:
Remove. It seems that you need to repeat/learn exceptions and what is their sense